### PR TITLE
Add PSP creation to prometheus config example

### LINKED ISSUE #484: [doc] 5.1.4  Prometheus
5.1.4  Prometheus

https://www.suse.com/documentation/suse-caasp-3/singlehtml/book-caasp-admin/book-caasp-admin.html#prometheus

Due to upstream changes in the Prometheus Helm Chart, the current steps to deploy Prometheus no longer work and need a slight update.

I was told "This seems to be due to Update psps to support all pods (#15055) · helm/charts@af1e617"
Related: https://github.com/helm/charts/commit/af1e617edb58566f859c8c3aabb509c3edafc85b#r34888663

Per a conversation on the CaaSP mailing list one change could be to add "--set podSecurityPolicy.enabled=true" to the end of the "helm install" line for Prometheus, like this:
helm install --name prometheus stable/prometheus --namespace monitoring --values prometheus-config-values.yaml --set podSecurityPolicy.enabled=true

The better/recommended option would be to add a top level PSP as well:
But for the latest chart version is should add the new top level setting as well:

podSecurityPolicy:
  enabled: true

### PR DESCRIPTION
**Pull request by r0ckarong**
Closes #484

### CODE REVIEW

File: xml/admin_monitoring.xml [hunk 1]
```diff
@@ -415,6 +415,10 @@ Adding password for user admin
       </listitem>
      </itemizedlist>
 <screen language="yaml">
+# Enable the creation of pod security policy
+podSecurityPolicy:
+  enabled: true
```
Comment (mjura) [inline xml/admin_monitoring.xml]: I think better would be to change it in our Prometheus Helm charts 
https://github.com/SUSE/kubernetes-charts-suse-com/blob/master/stable/prometheus/values.yaml#L5

Comment (mjura) [reply xml/admin_monitoring.xml]: I will take care of it

Comment (r0ckarong) [reply xml/admin_monitoring.xml]: This change is for v3 where we use the upstream prometheus chart. Is prometheus available from the suse registry for v3?

Comment (mjura) [reply xml/admin_monitoring.xml]: ahhh, I am sorry I didn't notice that it comes from v3. Yes, we are shipping now our own Helm charts and prometheus container in registry. Just wondering if we update documentation for v3 too.

**Comment by r0ckarong** [comment]
> This is CaaSP v3 documentation change and does not affecting v4
> +1

So what is there to do now? Is the change here OK for v3? Or do we need to change the docs to reflect usage of our own image for v3?

File: xml/admin_monitoring.xml (hunk 1 above)
Comment (mjura) [reply xml/admin_monitoring.xml]: I have submitted PR to SUSE Prometheus helm charts https://github.com/SUSE/kubernetes-charts-suse-com/pull/105

**Comment by mjura** [comment]
> > This is CaaSP v3 documentation change and does not affecting v4
> > +1
> 
> So what is there to do now? Is the change here OK for v3? Or do we need to change the docs to reflect usage of our own image for v3?

@kkaempf Could you please answer if we want waste some time on fixing CaaSP v3 documentation about using SUSE Prometheus and Grafana helm charts and images?

**Comment by kkaempf** [comment]
No. CaaSP v3 is in (reactive) maintenance. Bugfixing only.
SUSE-provided prometheus and grafana is v4-only.

**Pull request merged**